client/views/actions/action.tpl

```diff
@@ -1,2 +1,3 @@
-<span role="button" class="user {{colorClass from}}" data-name="{{from}}">{{mode}}{{from}}</span>
-<span class="action-text">{{{parse text}}}</span>
+<span role="button" class="user {{colorClass from}}" data-name="{{from}}">
+	{{~mode}}{{from~}}
+</span><span class="action-text">{{{parse text}}}</span>
```

Comment:
This needs a space between the 2 spans.

Currently showing up like this: `19:54 +puriburyshakes froakieworkie `